src/platform_impl/linux/x11/mod.rs

```diff
@@ -99,6 +100,11 @@ impl<T> PeekableReceiver<T> {
     }
 }
 
+#[derive(Clone)]
+pub struct OwnedDisplayHandle {
+    xconn: Arc<XConnection>,
```

Comment:
you can use `Rc` for it given that it's neither send nor sync.

Reply:
The object that's being passed around uses an `Arc` because it's being stored in a static variable.

https://github.com/rust-windowing/winit/blob/fbea75d31f4025825e9bf88b99d7c92e4eb79273/src/platform_impl/linux/mod.rs#L118-L120

If we want to transition away from this model, it'd probably need to take place outside of this PR.

Reply:
Hm, that's fair.